web-ui/src/pages/CertificationReportPage.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useReducer, useState } from 'react';
```

Comment:
This implements the new Certifications report page which provides three things:
- A button to open the "Manage Certifications" dialog.
- An "Earned Certifications table.
- The "Managed Certifications" dialog.